python_modules/dagster/dagster_tests/daemon_tests/test_backfill.py

```diff
@@ -2642,19 +2780,28 @@ def test_old_dynamic_partitions_job_backfill(
     assert instance.get_runs_count() == 4
 
 
+@pytest.fixture
+def instance_with_backfill_log_storage_enabled(instance):
```

Comment:
very nitty nit - it's nice when fixtures are all together at the top of a file

Reply:
This actually went in already in https://github.com/dagster-io/dagster/pull/26462; I can rebase and then, if you and @gibsondan want, happy to move it to the top. :)